week-3/B-array-some/exercise.js

```diff
@@ -11,6 +11,11 @@ var pairsByIndex = [[0, 3], [1, 2], [2, 1], null, [3, 0]];
 // If there is a null value in the array exit the program with the error code
 // https://nodejs.org/api/process.html#process_process_exit_code
 // process.exit(1);
+var containNull = pairsByIndex.some(function (include) {
+  include === null;
```

Comment:
I don't think this is quite right. We're not actually checking if the `include` variable is null. We need to use an `if` statement to do that.

Reply:
@40thieves Not quite sure how the process.exit(1), work...